src/main/java/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,74 @@
+package duke.commands;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import duke.Deadline;
+import duke.Event;
+import duke.Task;
+import duke.TaskList;
+import duke.ToDo;
+import duke.Ui;
+
```

Comment:
Maybe you could add some header comments to the class and the public methods? I noticed most of the Command files are missing these comments as well, perhaps because they are newly committed files

Reply:
You are right, thank you for pointing it out! I have only added header comments for half of my class and public methods and I am still working towards completing for all of them.